classes/core/PKPRequest.inc.php

```diff
@@ -591,6 +591,18 @@ function &getUser() {
 	function getUserVar($key) {
 		$_this = PKPRequest::_checkThis();
 
+		if (is_a($this, 'Request')) {
```

Comment:
With `$this` or with `$_this`? It was probably a static call (with `$this` remaining set from some other context).